src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageMetadataControl.xaml

```diff
@@ -163,9 +164,10 @@
         Margin="0,8,0,0"
         Text="{x:Static nuget:Resources.Label_DatePublished}" />
       <TextBox
+        Name="datePublished"
         Style="{DynamicResource SelectableTextBlockStyle}" 
         Visibility="{Binding Path=Published,Converter={StaticResource NullToVisibilityConverter}}"
-        Text="{Binding Path=Published,StringFormat={}{0:D} ({0:d})}"
+        Text="{Binding Path=Published,Converter={StaticResource DateFormatConverter}}"
```

Comment:
It might be worth verifying how the narrator narrates this format and if it passes the accessibility requirements